src/targets/__tests__/maven.test.ts

```diff
@@ -61,23 +61,108 @@ function getTargetOptions() {
   };
 }
 
+function getRequiredTargetConfig(): any {
+  return {
+    OSSRH_USERNAME: DEFAULT_OPTION_VALUE,
+    OSSRH_PASSWORD: DEFAULT_OPTION_VALUE,
+    gradleCliPath: DEFAULT_OPTION_VALUE,
+    mavenCliPath: DEFAULT_OPTION_VALUE,
+    mavenSettingsPath: DEFAULT_OPTION_VALUE,
+    mavenRepoId: DEFAULT_OPTION_VALUE,
+    mavenRepoUrl: DEFAULT_OPTION_VALUE,
+    android: false,
```

Comment:
My thinking about what this config field should answer is "I don't want to configure Android".

- I think of `null` as "there's no Android config". Why isn't there an Android config? Have you forgotten about it? This, to me, is like not answering a question.
- I think of `undefined` as "the Android config hasn't been defined". Why don't you define it then, if it's a config file? Do you want to set an `android` config or not? This, to me, is like giving a very ambiguous answer to a question.
- I think of `false` as "I don't want to set an Android config, but I'm saying it". This, to me, is like saying you don't want to set the configuration (answering with a nonambiguous answer).

Thus, using `false` seems the most appropriate for me. Do you have a different opinion about any (or all) of them? What do you think? Is it a best practice to use another type? (I might have thought about the field like `isAndroid` too.)

Reply:
I understand and mostly agree with the reasoning. But requiring to be explicit can lead to unnecessary frustration or repetition, especially in configurations like these. I'm almost thinking about it might be better to split `android` and `maven` targets and make them share a common core to address your concerns here.